app/components/orange-bar-custom-map-study.js

```diff
@@ -110,5 +111,21 @@ export default Component.extend({
 
       this.set('advanced', !this.get('advanced'));
     },
-  }
+    
+    toggleMapLayersMenu() {
+      this.get('metrics').trackEvent('GoogleAnalytics', {
+        eventCategory: 'Advanced Options',
+        eventAction: 'Toggle Map Layers Menu',
+        eventLabel: this.get('mapLayersMenu') ? 'Closed' : 'Opened',
+      });
+
+      this.set('mapLayersMenu', !this.get('mapLayersMenu'));
+    },
+    addedfile() {
```

Comment:
camelCase here

Reply:
Are these stubs for shapefile funcitons?

Reply:
They are stubs but I'll fix them in my upcoming shapefile PR